src/lib/global.svelte.ts

```diff
@@ -172,8 +214,33 @@ async function initRoomy(agent: Agent): Promise<Roomy> {
     ["authorization", token],
   );
 
-  // Use this instead of you want to test with a local development Leaf syncserver.
+  // // Use this instead of you want to test with a local development Leaf syncserver.
   // const websocket = new WebSocket("ws://127.0.0.1:8095");
+  let str = storage.getItem(g.channel.id) || "0";
+  let count = parseInt(str);
+  storage.setItem(g.channel.id, count);
+  setTimeout(() => {
+    websocket.addEventListener("message", async (event) => {
+      const decoded = decode(event.data);
+      const entityId = decoded.entityId;
+      const decoder = new TextDecoder("utf-8");
+      const update = decoder.decode(decoded.update);
+
+      // const items = await g.space?.sidebarItems.items();
+      // console.log("space sidebar items");
+      // for (const item of items!) {
+      //   let channel = item.tryCast(Channel);
+      //   console.log("channel", channel!.id);
+      // }
+      if (g.channel.id != entityId) {
+        count += 1;
+        // localStorage.setItem(`unreads:${entityId}`, unread_count.toString());
+        storage.setItem(entityId, count);
+      }
+      // console.log("channels", g.channel.id, entityId);
+      // console.log("message event", decoded, update);
+    });
+  }, 5000); // wait until initial page load is done, probably a better way to do this
```

Comment:
We made a change to the initial page load after logging in and setting up the web socket. I don't think you'll need this anymore if you catch up with main.

Reply:
this is part of a bigger refactor where we have to figure out how to listen to channels that aren't open